erpnext_autoinstall/install.py

```diff
@@ -3,14 +3,23 @@
 import frappe
 
 
+
+
 def after_install():
     """Frappe trigger after application install."""
+    configure_home_page()
     configure_website()
     configure_ldap()
     # FIXME: Will not "really" work since first run wizard will reset email settings...
     # configure_email()
 
 
+def configure_home_page():
+    home_page = os.getenv('HOME_PAGE', 'desk')
```

Comment:
I think it might be better to not have default value here and only apply the modification if the parameter is set.
This would prevent breaking ERPNext on future versions where home page link might change.